src/components/course/routes/ExternalCourseEnrollment.jsx

```diff
@@ -20,13 +25,37 @@ const ExternalCourseEnrollment = () => {
     },
     userSubsidyApplicableToCourse,
     hasSuccessfulRedemption,
+    externalCourseFormSubmissionError,
   } = useContext(CourseContext);
+  const {
+    enterpriseConfig: { authOrgId },
+  } = useContext(AppContext);
+
   const courseMetadata = useMinimalCourseMetadata();
 
+  const externalDashboardQueryParams = new URLSearchParams();
+  if (authOrgId) {
+    externalDashboardQueryParams.set('org', authOrgId);
+  }
+
+  let externalDashboardUrl = config.GETSMARTER_LEARNER_DASHBOARD_URL;
+  if (externalDashboardQueryParams.size > 0) {
+    externalDashboardUrl += `?${externalDashboardQueryParams.toString()}`;
+  }
+
   const {
     failureReason,
     failureMessage,
   } = useExternalEnrollmentFailureReason();
+
+  const containerRef = createRef(null);
+
+  useEffect(() => {
+    if (isDuplicateExternalCourseOrder(externalCourseFormSubmissionError) && containerRef?.current) {
+      containerRef?.current?.scrollIntoView({ behavior: 'smooth' });
```

Comment:
can probably defer on this test coverage that codecov is complaining about below.